### PR TITLE
fix: display 0 percent change on explore

### PR DESCRIPTION
**Pull request by cartcrom**
before:
image

after:
image

### CODE REVIEW

File: src/components/Tokens/TokenDetails/PriceChart.tsx [hunk 1]
```diff
@@ -55,17 +55,23 @@ export function calculateDelta(start: number, current: number) {
...
   }
+  return <StyledUpArrow size={16} key="arrow-up" />
 }
 
-export function formatDelta(delta: number) {
+export function formatDelta(delta: number | null | undefined) {
+  // Null-check not including zero
+  if (delta === null || delta === undefined) {
+    return '-'
+  } else if ((delta < 0.01 && delta > 0) || (delta > -0.01 && delta < 0)) {
```
Comment (vm) [inline src/components/Tokens/TokenDetails/PriceChart.tsx:72]: why is this necessary?

**Review by vm — APPROVED** [review]

File: src/components/Tokens/TokenDetails/PriceChart.tsx (hunk 1 above)
Comment (cartcrom) [reply src/components/Tokens/TokenDetails/PriceChart.tsx]: whoops meant to remove that part
